README.md

```diff
@@ -39,18 +39,28 @@ Set up project with Docker
     docker exec -it linkedevents-backend python manage.py event_import helmet --events
     # Import events from Espoo
     docker exec -it linkedevents-backend python manage.py event_import espoo --events
+    # Import organizations
+    docker exec -it linkedevents-backend python manage.py import_organizations https://api.hel.fi/paatos/v1/organization/ -s helsinki:ahjo
     ```
-6. Update search indexes:
+6. Add datasources helsinki and helfi:
```

Comment:
We will have to somehow automate this with @vikoivun ; while this is correct, it is not desired that the user will have to do stuff in the shell by hand.

Also, perhaps we want to make the default setup non-helsinki specific. However, the problem remains that the current UI is helsinki-specific and requires the data sources below, so we cannot solve this right away.